bot/__init__.py

```diff
@@ -41,3 +45,5 @@
     asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
 
 apply_monkey_patches()
+
+instance: "Bot" = None  # Global Bot instance.
```

Comment:
Why do we need this?

Reply:
We use it in a few places in bot, such as utils where we don't have access to the bot var very easily.

I've done this here mostly for consistency.

Reply:
https://sourcegraph.com/search?q=context:global+repo:python-discord.bot+%5Cbinstance%5Cb&patternType=regexp for examples

Reply:
Ah, I see